### ISSUE TITLE
Rotate a surface

### ISSUE THREAD

**Issue by olivercrush** [kr15h/ofxPiMapper#148]
Hi,

It's not really a bug, but I have to map vertical hexagons and the hexagon surface you generate is an horizontal one. I repositionned all the vertices to match my shape, but it rotates the image I used. I will solve this problem by using an already rotated image, but I wanted to know if there is a way to simply rotate the surfaces (like the size system with the mouse in the ProjectionMapping mode). And if not, if you planned to add this feature ?

Thank you !

**Comment by pietrondo** [comment]
good feature..

**Comment by kr15h** [comment]
Yes, this would be a good additional feature. Currently I am a bit off with tasks, but later this summer I am planning of activating the development a bit.
